.github/workflows/release.yml

```diff
@@ -38,6 +38,18 @@ jobs:
           SLACK_WEBHOOK: ${{ secrets.SEMANTIC_RELEASE_SLACK_WEBHOOK }}
         run: yarn semantic-release
 
+      - uses: masci/datadog@v1
+        with:
+          api-key: ${{ secrets.DATADOG_API_KEY }}
+          metrics: |
+            - type: "count"
+              name: "logrock.deploy"
+              value: 1.0
+              host: ${{ github.repository_owner }}
+              tags:
+                - "project:${{ github.repository }}"
+                - "env:production"
+
```

Comment:
can you place this after the storybook deploy? deploy of the lib requires npm publish + storybook documentation up to be considered complete.

Reply:
Ok, sure.